features/FxA-80-eye-password-experiment/README.md

```diff
@@ -0,0 +1,147 @@
+
+Eye Password Experiment
+========================
+
+https://mozilla.aha.io/features/FXA-80
+
+Firefox Account users are struggling immensely when changing their passwords.
+
+![Change Password Statistics](change-password-stats.png)
+
+[View Live Chart](https://app.datadoghq.com/graph/embed?token=f39f21f7fb65ebbd224d49e88dd365b817e869d68d2a9e384f56e908e0df666b&height=500&width=1000&legend=true)
+
+We would like to learn how we can help them struggle less by experimenting with different password unmasking treatments. We may use these results to inform a choice about unmasking passwords in platform.
+
+# Control
+Currently we use a simple “show” toggle. Both password field toggles are synchronized which we should remove this for the experiment (and permanently as filed in [#3600](https://github.com/mozilla/fxa-content-server/issues/3600)).
+
+<img alt="Current Masked" src="current-masked.png" width="525">
+
+<img alt="Current Unmasked" src="current-unmasked.png" width="525">
+
+# Treatments
+## No Show
+
+Remove the unmasking functionality from both new and old password fields.
+
+<img alt="No Show" src="eye-password-noshow.png" width="525">
+
+
+## Toggle
+
+Likely more usable, but more revealing, is the toggle treatment. We should run this interaction with all of the available designs. Deprecated eye in animation for position only.
+
+<img alt="Toggle" src="eye-password-toggle.gif" width="525">
+
+### Acceptance Criteria
+
+* When the user hovers over the control and the password is masked, there should be a tooltip that says "Show password"
+* When the user hovers over the control and the password is unmasked, there should be a (more subtle) link title on the control that says "Hide password"
+* When the user clicks the control and the password is masked, the password becomes unmasked.
+* When the user clicks the control and the password is unmasked, the password becomes masked.
+
+## Press & Hold Down
+
+Possibly more familiar to Windows users, we can only show the password when the user presses and holds down on the icon (deprecated eye in animation for position only)
+
+<img alt="Press and Hold Down" src="eye-password-press.gif" width="525">
+
+### Acceptance Criteria
+
+* When the user hovers over the control and the password is masked, there should be a tooltip on the eye that says "Show password".
+* Windows-style: Only while the user is clicking the eye, does the password become unmasked.
+
+## Unmasked by Default
+
+A radical proposal that we should probably only test with the **Checkbox**  design.
```

Comment:
@shane-tomlinson This is a tough one. If we use usertesting.com, they are certain that someone is watching (us) and they may enter their password (the same one they use everywhere) while being recorded. In person might be better. If either of you can deploy a special build with this design, I'll remove it from the experiment. Deal?